src/MCPServer/lib/linkTaskManagerChoice.py

```diff
@@ -57,6 +57,9 @@ def __init__(self, jobChainLink, pk, unit):
 
         choices = MicroServiceChainChoice.objects.filter(choiceavailableatlink_id=str(jobChainLink.pk))
         for choice in choices:
+            if (django_settings.DISABLE_SEARCH_INDEXING and
+                    choice.chainavailable.description == 'Send to backlog'):
```

Comment:
Could we instead have a number of abilities and hide these items based on that? E.g. pipeline `X` has the `search` ability disabled, choice `Y` depends on `search` ability so it's ignored.

Reply:
Yes, I think that's a good idea. I've tried to implement what you've described in the most recent commit. Let me know what you think.